File: front-end/src/app/service-models/app/v1/app-users/client-user-address-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';
import { ClientUserAddressTypeSM } from '../../enums/client-user-address-type-s-m.enum';

export class ClientUserAddressSM extends CoinManagementServiceModelBase<number> {
    country!: string;
    state!: string;
    city!: string;
    address1!: string;
    address2!: string;
    pinCode!: string;
    clientUserAddressType!: ClientUserAddressTypeSM;
    clientUserId!: number;
    clientCompanyDetailId!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/AppUsers/ClientUserAddressSM.cs
using Siffrum.Web.Payroll.ServiceModels.Enums;

namespace Siffrum.Web.Payroll.ServiceModels.v1.AppUsers
{
    public class ClientUserAddressSM : Base.SiffrumPayrollServiceModelBase<int>
    {
        public string Country { get; set; }
        public string State { get; set; }
        public string City { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string PinCode { get; set; }
        public ClientUserAddressTypeSM ClientUserAddressType { get; set; }
        public int ClientUserId { get; set; }
        public int ClientCompanyDetailId { get; set; }
    }
}
